lightning/src/ln/chanmon_update_fail_tests.rs

```diff
@@ -2365,8 +2366,9 @@ fn do_channel_holding_cell_serialize(disconnect: bool, reload_a: bool) {
 		// disconnect the peers. Note that the fuzzer originally found this issue because
 		// deserializing a ChannelManager in this state causes an assertion failure.
```

Comment:
Isn't quite clear what issue the fuzzer had found before and if it would still find it after moving the initialization of `chan_0_monitor_serialized`.